src/data/nat/totient.lean

```diff
@@ -172,6 +206,20 @@ by rcases exists_eq_succ_of_ne_zero (pos_iff_ne_zero.1 hn) with ⟨m, rfl⟩;
 lemma totient_prime {p : ℕ} (hp : p.prime) : φ p = p - 1 :=
 by rw [← pow_one p, totient_prime_pow hp]; simp
 
+/-- Any positive natural is a prime power or a product of coprime numbers -/
+lemma prime_power_or_mul_coprime {n : ℕ} :
+  (∃ a p : ℕ, p.prime ∧ n = p ^ a) ∨ (∃ a b : ℕ, 2 ≤ a ∧ 2 ≤ b ∧ a.coprime b ∧ n = a * b) :=
+begin
+  sorry,
+end
+
+lemma totient_prime_iff {p : ℕ} : p.prime ↔ φ p = p - 1 :=
```

Comment:
I PR'd this as #8833

Reply:
Excellent, thank you. I have merged your `totient_prime_iff` branch into this one.

Reply:
you should probably use the depend bot thing

Reply:
I don't know what you mean by "the depend bot thing", sorry.

Reply:
see what Yakov did in #8859, for example. you put the PRs you need in your description, and it won't let you merge until they are merged